TabGroup/TabGroup.js

```diff
@@ -73,7 +74,7 @@ const TabBase = kind({
 		 * @type {Function}
 		 * @public
 		 */
-		onClick: PropTypes.func,
+		onTabClick: PropTypes.func,
```

Comment:
Can't we keep the onClick prop name as it is?

Reply:
Please leave the "cause" of why "onClick" does not work.

Reply:
Can you check my comment below?